crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -239,7 +302,30 @@ async fn execute_test(
     let metadata_updated = || async {
         onchain.mint_block().await;
         let order = services.get_order(&uid).await.unwrap();
-        are_close_enough(order.metadata.executed_surplus_fee, expected_surplus_fee)
+        is_approximately_equal(order.metadata.executed_surplus_fee, expected_surplus_fee)
     };
     wait_for_condition(TIMEOUT, metadata_updated).await.unwrap();
 }
+
+pub enum FeePolicyKind {
```

Comment:
No need to make this public.
```suggestion
enum FeePolicyKind {
```

Reply:
fixed